### ISSUE TITLE
Privacy Policy - Send to non-job board imported candidates (JH)

### ISSUE THREAD

**Issue by DaiGumms** [DaiGumms/Data#543]
Morning All,

Just to clarify the position following discussions.

We need to ensure we issue a privacy notice to candidates added using this tool – I’m told this is relatively simple to do and can be available within a couple of weeks. Client data is being treated as out of scope – this may be a position that is revisited in the future as the industry and beyond adapts to new data protection rules but for now contacts added can be excluded from notification. On that basis there is no issue with the tool going live followed within a week or 2 by the notification process.

Some guidance would be useful for users to let them know the candidates they add will receive an automatic notification so, to prevent high volumes of deletion requests coming back, some form of interaction with the candidate before adding is ideal.

Once this is in place for new candidates a mop-up will be required for those already added (as you’ve mentioned Keith).

For future work related to data we’ll need to consider a data protection impact assessment to consider and agree these points prior to development.

Regards,

Jen

┆Attachments: <a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;8nNF6Wq6">https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;8nNF6Wq6</a>

**Comment by DaiGumms** [comment]
➤ Keith Henry commented:

This appears to be a duplicate of https://trello.com/c/8nNF6Wq6

Do we need a second card?
